app/assets/javascripts/votes/votes.js

```diff
@@ -0,0 +1,50 @@
+$(function(){
+  updateRibbonLinks();
+  $('.ribbon').on('click', function(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    var $this = $(this);
+    var $spanTag = $this.find('span');
+    var $aTag = $this.find('a');
+    var currentText = $spanTag.text();
+    var url = $aTag.attr('href');
+
+    if (currentText === 'Vote') {
+      var method = 'POST'
+      var newText = 'Voted!'
+    } else {
+      var method = 'DELETE'
```

Comment:
'method' is already defined.<br>Missing semicolon.<br>'method' is defined but never used.